w10h02/test/pgdp/teams/ArtemisTest.java

```diff
@@ -0,0 +1,94 @@
+
+    @Test
+    @DisplayName("computeScores: großes Beispiel")
+    public void computeScoresLarge() {
+        var lineup = new Lineup(Set.of(eve, hanna), Set.of(sachmi, jasmine), Set.of(enrico, jakob));
+
+        assertEquals(34505, lineup.getTeamScore());
+        assertEquals(34014, lineup.getTeamSkill());
+        assertEquals(491, lineup.getTeamSynergy());
+    }
+
+    @Test
+    @DisplayName("computeOptimalLineup: kleines Beispiel")
+    public void computeOptimalLineupSmall() throws NoSuchFieldException, IllegalAccessException {
+        var lineup = Lineup.computeOptimalLineup(Set.of(eric, nils, felix, thomas), 2, 1, 1);
+
+        var attackers = getField(lineup, "attackers");
+        var defenders = getField(lineup, "defenders");
+        var supporters = getField(lineup, "supporters");
+
+        assertEquals(2, attackers.size());
+        assertEquals(1, defenders.size());
+        assertEquals(1, supporters.size());
+
+        assertTrue(attackers.contains(nils));
+        assertTrue(attackers.contains(eric));
+        assertTrue(defenders.contains(felix));
+        assertTrue(supporters.contains(thomas));
+    }
+
+    @Test
+    @DisplayName("computeOptimalLineup: großes Beispiel [Die Laufzeit ist für diese Aufgabe nicht besonders wichtig. Solange das große Beispiel für computeOptimalLineup nicht timeoutet bist du auf der sicheren Seite. Lokal solltest du je nach PC für das Beispiel nicht viel mehr als 30 Sekunden benötigen.]")
+    // @Disabled // Comment this line in case you don't want to test this function.
+    public void computeOptimalLineupLarge() throws NoSuchFieldException, IllegalAccessException {
+        System.err.println("You are currently running `computeOptimalLineupLarge()` which can be CPU intensive.");
+        long timeBefore = System.currentTimeMillis();
+
+        var lineup = Lineup.computeOptimalLineup(
+                Set.of(jan, georg, anton, johannes, konrad, max, oliver, robin, laura, lukas), 2, 3, 5);
+
+        long timeAfter = System.currentTimeMillis();
+        long elapsedTime = timeAfter - timeBefore;
+
+        System.out.printf("Test `computeOptimalLineupLarge()` took %sms to execute.\n", elapsedTime);
+
+        var attackers = getField(lineup, "attackers");
+        var defenders = getField(lineup, "defenders");
+        var supporters = getField(lineup, "supporters");
+
+        assertEquals(2, attackers.size());
+        assertEquals(3, defenders.size());
+        assertEquals(5, supporters.size());
+
+        assertTrue(attackers.contains(max));
+        assertTrue(attackers.contains(georg));
+        assertTrue(defenders.contains(jan));
+        assertTrue(defenders.contains(anton));
+        assertTrue(defenders.contains(konrad));
+        assertTrue(supporters.contains(lukas));
+        assertTrue(supporters.contains(laura));
+        assertTrue(supporters.contains(johannes));
+        assertTrue(supporters.contains(oliver));
+        assertTrue(supporters.contains(robin));
```

Comment:
Set attackersExpected = Set.of(max,georg);
Set defendersExpected = Set.of(jan,anton,konrad);
Set supportersExpected = Set.of(lukas,laura,johannes,oliver,robin);

assertEquals(attackersExpected, attackers);
assertEquals(defendersExpected, defenders);
assertEquals(supportersExpected, supporters);